framework/src/application/node/node.ts

```diff
@@ -209,7 +209,7 @@ export class Node {
 				Object.assign(this._applicationState, event.data);
 			});
 
-			this._logger.info('Modules ready and launched');
+			this._logger.info('Node ready and launched');
```

Comment:
since process, sync initiated below, is this log line correct? maybe it should below that?

Reply:
I think its correct here, Node is ready and launched. Rest is its internal processing.